src/views/text_area.rs

```diff
@@ -362,6 +363,14 @@ impl TextArea {
 }
 
 impl View for TextArea {
+    fn required_size(&mut self, constraint: Vec2) -> Vec2 {
+        self.compute_rows(constraint);
+        Vec2::new(
+            self.rows.iter().map(|r| r.width).max().unwrap_or(1),
```

Comment:
I think it needs a `+1` here to work properly (we need 1 extra column for the cursor itself).